releasenotes/notes/add-alerts-for-swap-availability-75e28ed7f913d1ec.yaml

```diff
@@ -2,8 +2,8 @@
 features:
   - |
     Added two alerts (Warning and critical) that are triggered when the ratio
```

Comment:
nit: warning is capitalised but critical is not?
Not worth holding up this PR for but thought I'd mention it while we're here

Reply:
Fixed